### ISSUE TITLE
Closing the test pane leaves a blank bar

### ISSUE THREAD

**Issue by xtr3me** [moxley/atom-ruby-test#87]
When closing the test pane it leaves a blank bar on the screen. The blank bar stacks up, so when you're opening and closing the test pane you'll end up with multiple bars which consume your screen estate.
It doesn't matter if i close it with the cross icon or with ctrl+cmd+x shortcut

Atom version: 1.7.4
Seti-UI theme: 1.2.3
Seri-Syntax: 1.0.1

Re-installing the package doesn't work.

**Comment by moxley** [comment]
Is anyone else having this issue?

**Comment by barelyknown** [comment]
Yes, I've seen this issue. It doesn't happen every time the test pane is closed, but it happens regularly. Eventually I need to close the window and reopen to get back the full vertical screen estate.
